neo/rawio/axonarawio.py

```diff
@@ -377,8 +377,7 @@ def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop, strea
         if channel_indexes is None:
             channel_indexes = [i for i in range(bin_dict["num_channels"])]
         elif isinstance(channel_indexes, slice):
-            channel_indexes_all = [i for i in range(bin_dict["num_channels"])]
-            channel_indexes = channel_indexes_all[channel_indexes]
+            channel_indexes = self.get_active_channels()
```

Comment:
```suggestion
            channel_indexes = self._get_active_channels()
```

just going to fix this myself really quick. We need to call the private method since we switched it to private :)